### ISSUE TITLE
Did we specifically discuss estimates of 2014 recruitment with Phi=0.21?

### ISSUE THREAD

**Issue by andrew-edwards** [cgrandin/hake-assessment#418]
This is related to Ian's "Last year we made a range of arguments..." comment just now in #406.

We don't really mention it in the text (I thought we had but can't find it), but Figure 50 shows that the yellow (Phi=0.21) 2014 recruitment is not much larger than our based one.  

Just now I was originally looking at the final variable recruitment year in Fig 50 thinking it is 2014 but it is actually 2016 - it has the yellow somewhat higher. So I thought we might be getting a bit circular - if we'd stuck with Phi=0.21 then 2014 would still be high. 

So Phi=0.21 does give higher recruitment for 2016 this time. Just mentioning this in case people start looking at Figure 50 and get confused because of the projected recruitments that shouldn't really be there (as we've discussed elsewhere). So maybe we should add a bit more text as well.

**Comment by taylori** [comment]
Good point about the circular argument. Looking at Figure 50, yes 2014 would be higher with Phi = 0.21, but not nearly as high as last years model predicted for the equivalent case (phi = 0.03).  Figure 51 also shows that the Phi = 0.21 model would have worse fit to the survey index.

We don't have a good way to compare the fit to the age-1 index for sensitivity analyses. In the future, we could keep the age-1 index in all models, but by default turn off its likelihood contribution by setting the fleet number to be negative. That would allow us to get comparison plots of the fit to the index for multiple models. But clearly the 3x higher 2014 vs 2010 produced last year was a bad fit, and likewise the yellow point for the 2016 cohort in Figure 50 shows that the relative size vs 2014 or 2010 wouldn't fit well to the age-1 index.

We discuss the situation last year in the 2nd paragraph under "2.4.3 Variability on fishery selectivity deviations" on page 40, but not the 2014 cohort in the current model. It's probably worth adding a sentence about it somewhere either there or in the sensitivity analysis section.

Finally, I just committed a fix to the r4ss plot that allowed forecast recdevs to be shown in Fig 50 among other places: https://github.com/r4ss/r4ss/issues/210. Re-installing r4ss via 
```
devtools::install_github('r4ss/r4ss', ref='development')
```
should fix those plots and make it more clear that the final large recruit is 2016 not 2014.

**Comment by andrew-edwards** [comment]
Thanks Ian. I'll do a checklist (ooh!) to make sure we all (at some point) update r4ss:

- [ ]  Ian
- [ ]  Chris
- [ ] Aaron
- [ ] Andy
- [ ] Add a sentence to text (see above)

So we can close Issue once all checked off I think.

**Comment by cgrandin** [comment]
I re-built the doc and Figure 50 now ends on 2018 so yes I can confirm the change works.
Thanks Ian

**Comment by taylori** [comment]
Good point @cgrandin. Looks like Figures 32 and 33 show recruitment through 2017, while Figures 15, 16, 50, 55, and 57 all include 2018. 

I'm not sure which is best, but probably would be good to make them consistent. It makes sense to report spawning biomass for 2018 since that's the basis for the harvest control rule, etc. And Tables 18-21 show 2018 recruitment. So perhaps it would be more consistent to add 2018 to Figures 32 and 33 rather than take it off of the other figures.

I'm OK either way, and obviously it isn't too big a deal since nobody noticed until this year.

**Comment by andrew-edwards** [comment]
I kind of think recruitments shouldn't really go to 2018, but then again there is a stock-recruit curve so it makes a little sense to have it. Table 18 shows MLEs for 2017 and 2018 recruits are 1036 and 1035 as essentially no information. But hot having them to 2020ish is better.

**Comment by aaronmberger** [comment]
I fully agree with always including the age-1 index in our models and just turning on/off that likelihood component.

**Comment by taylori** [comment]
This could be either left as a new issue with a 2019 benchmark or just added to the list in Andy's day planner which will be added back after the 2018 assessment is complete.

**Comment by andrew-edwards** [comment]
Leave this open so we add a sentence to the text (see start of Issue). We can do it during SRG meeting.

**Comment by andrew-edwards** [comment]
Screw it, not bothering to add text.
